lib/types/index.js

```diff
@@ -238,8 +237,6 @@ export type CustomFile = {
   useDuringServe: boolean
 }
 
-// TODO: Remove this eslint rule once https://github.com/babel/babel-eslint/pull/584
-// is merged in.
 /* eslint-disable no-use-before-define */
```

Comment:
I think we have an old version of flow since I still go the pre-commit lint error when I removed this

Reply:
Ok we should change the TODO then to be updating eslint rather than waiting for the PR to be merged. I think the TODO is still relevant as long as the error is thrown